markets/perps-market/test/integration/Account/Margins.test.ts

```diff
@@ -0,0 +1,164 @@
+      await openPosition({
+        systems,
+        provider,
+        trader: trader1(),
+        accountId,
+        keeper: trader1(),
+        marketId: perpsMarkets()[0].marketId(),
+        sizeDelta: bn(-2),
+        settlementStrategyId: perpsMarkets()[0].strategyId(),
+        price: bn(30_000),
+      });
+      await openPosition({
+        systems,
+        provider,
+        trader: trader1(),
+        accountId,
+        keeper: trader1(),
+        marketId: perpsMarkets()[1].marketId(),
+        sizeDelta: bn(20),
+        settlementStrategyId: perpsMarkets()[1].strategyId(),
+        price: bn(2_000),
+      });
+    });
+
+    let startingMargin: Wei,
+      initialPnl: Wei,
+      btcInitialMargin: Wei,
+      ethInitialMargin: Wei,
+      btcMaintenanceMargin: Wei,
+      ethMaintenanceMargin: Wei,
+      minimumPositionMargin: Wei;
+    before('identify expected values', () => {
+      startingMargin = wei(100_000);
+      const btcSkewScale = wei(perpsMarketConfig[0].fundingParams.skewScale);
+      const ethSkewScale = wei(perpsMarketConfig[1].fundingParams.skewScale);
+      const btcPnl = calculatePricePnl(wei(0), btcSkewScale, wei(-2), wei(30_000));
+      const ethPnl = calculatePricePnl(wei(0), ethSkewScale, wei(20), wei(2000));
+      initialPnl = btcPnl.add(ethPnl);
+
+      const notionalBtcValue = wei(2).mul(wei(30_000));
+      const notionalEthValue = wei(20).mul(wei(2000));
+
+      btcInitialMargin = notionalBtcValue.mul(wei(2).div(wei(btcSkewScale)).mul(wei(2)));
+      ethInitialMargin = notionalEthValue.mul(wei(20).div(wei(ethSkewScale)).mul(wei(2)));
+
+      // maintenance margin ratio == 1
+      btcMaintenanceMargin = notionalBtcValue.mul(wei(2).div(wei(btcSkewScale)));
+      ethMaintenanceMargin = notionalEthValue.mul(wei(20).div(wei(ethSkewScale)));
+
+      // in above config: 1000 + 500
+      minimumPositionMargin = wei(1500);
+    });
+
+    it('has correct available margin', async () => {
+      assertBn.equal(
+        await systems().PerpsMarket.getAvailableMargin(accountId),
+        startingMargin.add(initialPnl).toBN()
+      );
+    });
+
+    it('has correct withdrawable margin', async () => {
+      assertBn.equal(
+        await systems().PerpsMarket.getWithdrawableMargin(accountId),
+        startingMargin
+          .add(initialPnl)
+          .sub(btcInitialMargin)
+          .sub(ethInitialMargin)
+          .sub(minimumPositionMargin)
+          .toBN()
+      );
+    });
+
+    it('has correct initial and maintenance margin', async () => {
+      const [initialMargin, maintenanceMargin] = await systems().PerpsMarket.getRequiredMargins(
+        accountId
+      );
+      assertBn.equal(
+        initialMargin,
+        btcInitialMargin.add(ethInitialMargin).add(minimumPositionMargin).toBN()
+      );
+      assertBn.equal(
+        maintenanceMargin,
+        btcMaintenanceMargin.add(ethMaintenanceMargin).add(minimumPositionMargin).toBN()
```

Comment:
Same as before, I'm not sure why we would add the minimum margin here. Instead I would expect to test a situation where the required margin is _below_ the minimum, but this function returns the minimum.